packages/salesforce-adapter/test/change_validators/flow_referenced_elements.test.ts

```diff
@@ -220,31 +390,72 @@ describe('flowReferencedElements change validator', () => {
               connector: { [TARGET_REFERENCE]: 'ActionCall' },
             },
           ],
+          constants: [{ name: 'Const' }],
+          exitRules: [
+            { conditions: [{ [LEFT_VALUE_REFERENCE]: 'Const', rightValue: { [ELEMENT_REFERENCE]: 'ActionCall' } }] },
+          ],
+          actionCalls: [
+            {
+              name: 'ActionCall1',
+              inputParameters: [{ name: 'Param', value: { [ELEMENT_REFERENCE]: 'ActionCall.Param' } }],
+              connector: { [TARGET_REFERENCE]: 'ActionCall2' },
+            },
+            {
+              name: 'ActionCall2',
+              connector: { [TARGET_REFERENCE]: 'ActionCall1' },
+            },
+          ],
         },
-        Flow,
+        flow,
       )
-      flowChange = toChange({ after: flow })
+      flowChange = toChange({ after: flowInstance })
     })
     it('should create change error per reference', async () => {
       const errors = await flowReferencedElements([flowChange])
       expect(errors).toIncludeSameMembers([
         {
-          severity: 'Error',
+          severity: 'Warning',
           message: 'Reference to missing Flow Element',
           detailedMessage: `The Flow Element "${'ActionCall'}" does not exist.`,
-          elemID: flow.elemID.createNestedID('assignments', '0', 'connector', TARGET_REFERENCE),
+          elemID: flowInstance.elemID.createNestedID('assignments', '0', 'connector', TARGET_REFERENCE),
         },
         {
-          severity: 'Error',
+          severity: 'Warning',
           message: 'Reference to missing Flow Element',
           detailedMessage: `The Flow Element "${'ActionCall'}" does not exist.`,
-          elemID: flow.elemID.createNestedID('decisions', '0', 'connector', TARGET_REFERENCE),
+          elemID: flowInstance.elemID.createNestedID('decisions', '0', 'connector', TARGET_REFERENCE),
         },
         {
           severity: 'Info',
           message: 'Unused Flow Element',
           detailedMessage: `The Flow Element "${'Decision'}" isn’t being used in the Flow.`,
-          elemID: flow.elemID.createNestedID('decisions', '0', FLOW_NODE_FIELD_NAMES.NAME),
+          elemID: flowInstance.elemID.createNestedID('decisions', '0', FLOW_NODE_FIELD_NAMES.NAME),
+        },
+        {
+          severity: 'Warning',
+          message: 'Reference to missing Flow Element',
+          detailedMessage: `The Flow Element "${'ActionCall'}" does not exist.`,
+          elemID: flowInstance.elemID.createNestedID(
+            'exitRules',
+            '0',
+            'conditions',
+            '0',
+            'rightValue',
+            ELEMENT_REFERENCE,
+          ),
+        },
+        {
+          severity: 'Warning',
+          message: 'Reference to missing Flow Element',
+          detailedMessage: `The Flow Element "${'ActionCall'}" does not exist.`,
```

Comment:
```suggestion
          detailedMessage: 'The Flow Element "\'ActionCall\'" does not exist.',
```
nit what is this weird syntax?

Reply:
You are right, it's unnecessary, I changed it to:
` detailedMessage: 'The Flow Element "ActionCall" does not exist.',`